docs/topics/secured-mission-integration-tests.adoc

```diff
@@ -0,0 +1,39 @@
+= Running Integration Tests
+
+To run the integration tests included with this booster you must:
+
+* be connected to an OpenShift instance
+* navigate to the project that will be used for testing
+
+To run the integration tests:
+
+. Deploy the {RHSSO} server:
++
+[source,bash,options="nowrap",subs="attributes+"]
+--
+oc apply -f service.sso.yaml
+--
++
+. Execute the following command to run the tests. Use the following syntax to pass in the URL of the {RHSSO} server as a parameter:
++
+[source,bash,options="nowrap",subs="attributes+"]
+--
+$ mvn clean verify -Popenshift,openshift-it -DSSO_AUTH_SERVER_URL=$(oc get route secure-sso -o jsonpath='https://{.spec.host}/auth')
+--
++
+[WARNING]
+--
+Ensure that you add view access rights for the service account before executing tests.
+
+[source,bash,options="nowrap",subs="attributes+"]
+----
+$ oc policy add-role-to-user view -n $(oc project -q) -z default
```

Comment:
Not sure if this is really needed.

Reply:
I've been wondering about that one, too. I see that the callout about view access is included for configmap. I don't believe that it is necessary for the Secured boosters, though. I'll remove it